tests/test_runner/test_discover_runner.py

```diff
@@ -71,6 +79,65 @@ def test_pattern(self):
 
         self.assertEqual(count, 1)
 
+    @skipUnless(PY37, "unittest -k option requires Python 3.7 and later")
```

Comment:
Use single quotes (unless a string contains a single quote) as described in [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).